docs/sources/api/_index.md

```diff
@@ -308,6 +308,24 @@ And `<stream value>` is:
 
 See [statistics](#statistics) for information about the statistics returned by Loki.
 
+### Batching
```

Comment:
this seems to be removed? so loki does not support pagination at all in the future?